apps/ejabberd/src/mod_muc_light_admin.erl

```diff
@@ -210,10 +210,12 @@ is_subdomain(Child, Parent) ->
 
 iq(S, R, T, C) when is_binary(S),
                     is_binary(R), is_binary(T), is_list(C) ->
+    UUID = uuid:uuid_to_string(uuid:get_v4(), binary_standard),
```

Comment:
I'm actually for UUID as it is not only var name but also shortcut for `Universal Unique Identifier` which in many places is referred as UUID. Also to me `UUID` is more readable then `Uuid`.